lib/cucumber/core/test/result.rb

```diff
@@ -16,7 +16,7 @@ def self.query_methods(result_type)
               result_type
             end
 
-            [:passed, :failed, :undefined, :unknown, :skipped, :pending].each do |possible_result_type|
+            [:passed, :failed, :undefined, :unknown, :skipped, :pending, :ambiguous].each do |possible_result_type|
```

Comment:
These symbols definitions has been moved to a module level constant, so `:ambiguous` should be added there instead, and since the order control the order in the summary listing, I think is should be defined directly after `:failed`